forge/test/create_model_structure.sh

```diff
@@ -0,0 +1,160 @@
+# SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# !/bin/bash
+
+# : << 'COMMENT'
+# SAMPLE:
+# Script run command : bash forge/test/create_model_structure.sh
+
+# INPUTS:
```

Comment:
Let's create a doc section for this script as well.

Reply:
Sure Let me create a issue for adding documenting the script.

Reply:
Created a issue for the documenting the script - https://github.com/tenstorrent/tt-forge-fe/issues/801